test/e2e/tests/test_certificate.py

```diff
@@ -172,4 +223,63 @@ def test_crud_public(
 
         k8s.delete_custom_resource(ref)
         time.sleep(DELETE_WAIT_AFTER_SECONDS)
-        certificate.wait_until_deleted(certificate_arn)
\ No newline at end of file
+        certificate.wait_until_deleted(certificate_arn)
+
+    def test_import_certificate(
+            self,
+            certificate_import,
+    ):
+        (ref, cr) = certificate_import
+        assert k8s.wait_on_condition(
+            ref,
+            condition.CONDITION_TYPE_RESOURCE_SYNCED,
+            "True",
+            wait_periods=MAX_WAIT_FOR_SYNCED_MINUTES,
+        )
+        assert k8s.get_resource_condition(ref, condition.CONDITION_TYPE_TERMINAL) is None
+
+        assert 'status' in cr
+        status = cr['status']
+        assert 'ackResourceMetadata' in status
+        assert 'arn' in status['ackResourceMetadata']
+        certificate_arn = status['ackResourceMetadata']['arn']
+        assert status['type_'] == 'IMPORTED'
+        assert status['status'] == 'ISSUED'
+        assert status['subject'] == 'O=ACK,CN=services.k8s.aws'
+
+        assert certificate.get(certificate_arn) is not None
+
+        updates = {
+            'spec': {
+                'options': {
+                    'certificateTransparencyLoggingPreference': 'ENABLED'
+                }
+            },
+        }
+        k8s.patch_custom_resource(ref, updates)
+        time.sleep(10)
+        assert k8s.wait_on_condition(
+            ref,
+            condition.CONDITION_TYPE_TERMINAL,
+            'True',
+            wait_periods=MAX_WAIT_FOR_SYNCED_MINUTES,
+        )
```

Comment:
Why does the resource go in terminal state when `certificateTransparencyLoggingPreference` is enabled?

Reply:
It's because `certificateTransparencyLoggingPreference` is an immutable field for imported certificates.